apps/SageAccountWeb/index.html

```diff
@@ -21,32 +21,7 @@
             integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
             crossorigin="anonymous"
     />
-    <link
-            rel="stylesheet"
-            href="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.css"
-            integrity="sha384-9eLZqc9ds8eNjO3TmqPeYcDj8n+Qfa4nuSiGYa6DjLNcv9BtN69ZIulL9+8CqC9Y"
-            crossorigin="anonymous"
-    />
```

Comment:
KaTeX CSS is not bundled with SRC CSS (it includes >1MB of fonts), but we can still pull it from NPM